phpBB/phpbb/notification/type/mention.php

```diff
@@ -0,0 +1,406 @@
+<?php
+/**
+*
+* This file is part of the phpBB Forum Software package.
+*
+* @copyright (c) phpBB Limited <https://www.phpbb.com>
+* @license GNU General Public License, version 2 (GPL-2.0)
+*
+* For full copyright and license information, please see
+* the docs/CREDITS.txt file.
+*
+*/
+
+namespace phpbb\notification\type;
+
+/**
+* Post notifications class
+* This class handles notifications for replies to a topic
+*/
+
+class mention extends \phpbb\notification\type\base
```

Comment:
You could extend `post` instead. That would probably save a lot of work and maintenance in the future.